Userland/Games/Solitaire/Game.cpp

```diff
@@ -410,7 +420,7 @@ void Game::draw_cards()
         else
             update(play_bounding_box);
     }
-        
```

Comment:
This whitespace change is incorrectly added in the third commit and then fixed in the fourth - please don't make it in the first place to keep the diffs clean.

Reply:
Sorry! I only noticed the whitespace when the CI checks failed, and then I fixed it in a rush, and shoved it into the last commit without thinking. I've fixed that now.